HackathonSampleApp/AppDelegate.swift

```diff
@@ -15,6 +15,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: HackathonSampleApp/AppDelegate.swift;
  Line: 16;
-->

- :warning: Line should be 120 characters or less: currently 145 characters (`line_length`)